src/features/MentorPage/components/MentorList/MentorCard/Expanded/Languages.tsx

```diff
@@ -28,7 +28,7 @@ export const Languages = ({
 };
 
 const Header = styled(Text)`
-  margin: 3rem 0 0 0;
+  margin-top: 3rem;
```

Comment:
There are multiple changes for components that also affect the Mentors-page.
I ask if you have taken this into consideration when developing, so also looking at the Mentors-page in different resolutions to see how it looks still there?

Reply:
Yeah I tried to be careful with this and simplify/streamline the common components we use.